vtysh/vtysh.c

```diff
@@ -3058,6 +3058,24 @@ DEFUN (vtysh_copy_running_config,
 	return vtysh_write_memory(self, vty, argc, argv);
 }
 
+DEFUN (vtysh_copy_to_running,
+       vtysh_copy_to_running_cmd,
+       "copy FILENAME running-config",
+       "Apply a configuration file\n"
+       "Configuration file to read\n"
+       "Apply to current configuration\n")
+{
+	int ret;
+	const char *fname = argv[1]->arg;
+
+	ret = vtysh_read_config(fname);
+
+	/* Return to enable mode - the 'read_config' api leaves us up a level */
+	vtysh_execute_no_pager("enable");
+
+	return ret;
+}
+
```

Comment:
This is to read the file and stay on the vtysh command line right? Like the one proposed in https://github.com/FRRouting/frr/pull/5697 ?

Reply:
heh - Quentin was kind enough to propose 5697 when I first asked a question about this topic, but in the end he gave up and closed 5697...
I don't _mind_ the '-f' option :  it just seems dumb to have to run vtysh twice, or to exit and rerun it if there's some config snippet I want to pull in. so I've tried this approach to see if I can avoid the issues that killed 5697, while still getting something that is a familiar feature in many router OSes...

Reply:
Sure and thanks for explanation. I have checked the patch it works fine.